src/interfaces/todoItem-interface.ts

```diff
@@ -0,0 +1,43 @@
+import { FilingTypes, PaymentMethod } from '@/enums'
+import { PaymentErrorIF } from '@/interfaces'
+
+/** A Todo List item. */
+export interface TodoItemIF {
```

Comment:
A "todo item" is what is in our local list and displayed in the Todo List.

Conversely, a "task item" is what got from the Lear "tasks" endpoint (ie, API object).